src/screens/documentDetails/components/documentAttachments/index.tsx

```diff
@@ -1,24 +1,77 @@
-import Grid from "@mui/material/Grid";
-import {Button, Card, CardContent, Chip, Stack} from "@mui/material";
-import React from "react";
+import Grid from '@mui/material/Grid';
+import { Card, CardContent, Chip, Stack } from '@mui/material';
+import React, { useState, ChangeEvent, createRef } from 'react';
+import LoadingButton from '@mui/lab/LoadingButton';
+import SendIcon from '@mui/icons-material/Send';
 
 function DocumentAttachments() {
-    return (
-        <Grid container spacing={4}>
-            <Grid item md={10}>
-                <Card>
-                    <CardContent>
-                        <Chip label="Atasamente" color="primary" size="medium" sx={{mb: 3}}/>
-                    </CardContent>
-                </Card>
-            </Grid>
+  const [document, setDocument] = useState<File | null>(null);
+  const [uploading, setUploading] = useState<boolean>(false);
+  const documentInput = createRef<HTMLInputElement>();
 
-            <Grid item md={2}>
-                <Stack gap={4}>
-                    <Button variant='contained'>Ataseaza document</Button>
-                </Stack>
-            </Grid>
-        </Grid>);
+  const handleFileSelect = async (e: ChangeEvent<HTMLInputElement>) => {
+    if (documentInput.current) {
+      const inputFile = documentInput.current;
+      if (inputFile.files?.[0]) setDocument(inputFile.files?.[0]);
+      if (e.target.files?.[0]) {
+        setUploading(true);
+        const formData = new FormData();
+        formData.append('file', document as File);
+        try {
+          // TODO: replace dummy endpoint with real one
+          const response = await fetch('https://httpbin.org/post', {
+            method: 'POST',
+            body: formData,
+            headers: {
+              'Content-Type': 'multipart/form-data',
+            },
+          });
+          if (!response.ok) {
+            throw new Error(response.statusText);
+          }
+          console.log('Document atasat!');
+        } catch (error) {
+          console.error('Eroare in atasarea documentului: ', error);
```

Comment:
- replace this fetch with a method from documentApiService
- use the axios instance there for the server url with the appropriate path eg: `this.httpClient.post('/document/upload')`
- remove `if (!response.ok) {` and move `throw new Error(response.statusText);` in `catch` 
- remove console.logs